process/project_proposals.adoc

```diff
@@ -10,30 +10,18 @@ image::project-stages.png[Project stages]
 
 === Sandbox process
 
-_Note: we are currently experimenting with a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals]. If you are interested in submitting a project for Sandbox we recommending reaching out to CNCF staff, or raising in the TOC channel on CNCF Slack, for advice on how to proceed in the short term._
+Note: We have changed the sandbox proposal process to a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals].
 
 All exceptions (and "no" outcomes) are handled by the TOC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.
 
 image::sandbox-process.png[Sandbox process]
-
 . *Project Proposal*
-   * Project proposed through a https://github.com/cncf/toc/issues[GitHub issue]
-   * Template TBD
-. *TOC Triage* _2 weeks for a straightforward referral to a SIG; decline or defer could require more discussion_
-   * Performed by any TOC member
-   * Lazy TOC consensus before declining outright or deferring assessment at this time
-   * Picks appropriate https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md[CNCF SIG] to perform assessment
-. *SIG Assessment* _1-2 months_
-   * Project presents to SIG
-   * Lightweight - this is pre-DD
-   * Confirms that project accepts the https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#sandbox-stage[Sandbox entry requirements]
-   * SIG makes recommendation to TOC: Sandbox / Incubation / Decline 
+   * Project proposed through a https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit[form]
+   * Project proposed for contribution through a https://github.com/cncf/toc/pulls[Pull request] 
 . *TOC Review*
-   * TOC review recommendation and presentation
+   * TOC reviews proposal in spreadsheet and presentation, project passes for inclusion with a simple majority vote. 
```

Comment:
I think we can remove "and presentation" since that is optional

Reply:
```suggestion
   * TOC review Sandbox proposals in spreadsheet in a meeting every two months 
   * TOC use the following criteria:
      .. Is the project a fit for the CNCF, according to the charter and TOC principles, in the TOC’s opinion? 
      .. Is the project’s roadmap in line with the goals of the CNCF as defined in the charter and TOC principles? 
      .. Does the project appear to be on a good path to becoming well-governed and vendor-neutral? 
   * TOC votes on each submission, and projects will be accepted by a simple majority vote of the TOC. 
   * In the event of rejection, we will indicate which of these criteria the TOC did not believe were met. Projects are welcome to address feedback and submit again in future. Unless advised otherwise, projects should not submit again for six months. 
```

Adding detail on the criteria we use, taken from the Sandbox proposal google doc